### ISSUE TITLE
⌨ Notifications pane long list

### ISSUE THREAD

**Issue by vd1992** [GCTC-NTGC/gc-digital-talent#10638]
## ⌨ Accessibility Issue

When tabbing through a long list, it shifts the screen down but when you reach the end it tabs back to the top but doesn't scroll up so you no longer know what you have selected

This is when you've highlighted the "X", tabbing again moves the screen up

### WCAG 2.1 Criteria

List any relevant criteria that is failing.

## 🦋 Expected Behaviour

Scrolling up

## 📋 Steps to Reproduce

1. Have a lot of notifications
2. Tab through, follow the screen and focus

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7a774a9a-4447-42f2-a901-595802fb402d)

Where is the focus now???

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/3c7574b5-9f42-4a27-8261-778bcfa61a07)

Scroll up, oh there it is

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/2df15300-8bf7-49e0-b0b5-2189270a17f9)

## 🙋‍♀️ Proposed Solution

(optional) Do you have a proposed solution?

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Page scroll should follow the tab focus back up to the top X
